spring-integration-core/src/test/java/org/springframework/integration/config/InvalidChannelWithCapacityAndQueueParserTests-context.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans:beans xmlns="http://www.springframework.org/schema/integration" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:beans="http://www.springframework.org/schema/beans"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
+			http://www.springframework.org/schema/beans/spring-beans.xsd
+			http://www.springframework.org/schema/integration
+			http://www.springframework.org/schema/integration/spring-integration.xsd">
+
+	<channel id="output">
+		<queue ref="queue" capacity="10" />
```

Comment:
Agree, I did realize that for my new files an error message had appear, but not in yours (for `InvalidChannelWithMessageStoreParserTests-context`)